test/src/hitbox_test.cpp

```diff
@@ -23,3 +24,24 @@ TEST (HitboxTest, simple_hitbox_overlapping_false) {
     bool actual = hitbox1.is_overlapping(hitbox2);
     EXPECT_EQ(expected, actual);
 }
+
+TEST (HitboxTest, constructor_1) {

```

Comment:
The names could be a bit more descriptive. So instead of `constructor_1` which doesn't really say which constructor, you could write `member_constructor` and `sdl_rect_constructor`.